src/Features/Blockcore.Features.Wallet/Types/MultisigScheme.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Newtonsoft.Json;
+
+namespace Blockcore.Features.Wallet.Types
+{
+    public class MultisigScheme
+    {
+        /// <summary>
+        /// How many signatures will be suffient to move the funds.
+        /// </summary>
+        [JsonProperty(PropertyName = "threashold")]
+        public int Threashold { get; set; }
+
+        /// <summary>
+        /// Cosigner extended pubkeys. Intentionally not including any xPriv at this stage as such model is simplest to start with.
+        /// </summary>
+        [JsonProperty(PropertyName = "xPubs")]
+        public string[] XPubs { get; set; }
```

Comment:
You may want to add also the trx hash that is being signed and list of sigs?

Reply:
This is a helper class that provides some fixed inputs for generation of entire HD multisig wallet. So there will be many addresses and many transactions as children in each address. If these are needed then they probably need to be composite children instead of belonging to this class?